corehq/apps/hqwebapp/management/commands/migrate_app_to_bootstrap5.py

```diff
@@ -220,9 +250,48 @@ def save_split_templates(original_path, bootstrap3_path, bootstrap3_lines, boots
         with open(bootstrap5_path, 'w') as file:
             file.writelines(bootstrap5_lines)
 
-    def refactor_references(self, old_reference, new_reference, is_template):
-        self.stdout.write("updating references...")
-        found_references = False
+    def _get_migrated_files(self, app_name):
+        app_template_folder = self._get_app_template_folder(app_name)
+        migrated_files = [f for f in app_template_folder.glob('**/*')
+                          if f.is_file() and '/bootstrap3/' in str(f) and '/crispy/' not in str(f)]
+        app_static_folder = self._get_app_static_folder(app_name)
+        migrated_files.extend(
+            [f for f in app_static_folder.glob('**/*.js')
+             if f.is_file() and '/bootstrap3/' in str(f)]
+        )
+        return migrated_files
+
+    def verify_migrated_references(self, app_name):
+        migrated_files = self._get_migrated_files(app_name)
+        template_path = str(self._get_app_template_folder(app_name))
+        for file_path in migrated_files:
+            is_template = template_path in str(file_path)
```

Comment:
Pathlib has a method to check if a path is relative to another:
```suggestion
        template_path = self._get_app_template_folder(app_name)
        for file_path in migrated_files:
            is_template = file_path.is_relative_to(template_path)
```